changelog.d/2828.change.rst

```diff
@@ -0,0 +1,2 @@
+Write long description in message payload of PKG-INFO file.
+Changed in metadata version 2.1
```

Comment:
@cdce8p please take a look at the explanations in the README here https://github.com/pypa/setuptools/tree/main/changelog.d — it's better to use the past tense consistently (your first sentence is in the present tense, in imperative form). You may optionally sign this fragment with your handle as shown in the examples.

Reply:
Using something else then present tense doesn't make sense IMO. Take `Written long description ...`, for example. This would be wrong since it's a continuous process and doesn't stop in the past.

Past tense does make sense when writing about a feature that has been added.